kato/kato-azure/kato-azure.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
```

Comment:
I not sure we need to differentiate kato from mort or oort. Those concepts are historical in nature and no longer serve a purpose. 

@duftler is going to do a bit of a sanity check to double check that merging kato-xxx/mort-xxx/oort-xxx into clouddriver-xxx doesn't cause (many) issues.

It's not something that gates this PR from going down, but should influence how we move forward.

I'd like to find some time to migrate the AWS components out (cc: @cfieber)